### PR TITLE
Create noise cancelation.md

### PR DESCRIPTION
**Pull request by Slayde00**

### CODE REVIEW

**Review by JesseTNRoberts — COMMENTED** [review]
I don't think the purpose of this subsystem is sufficiently explained. This reads like a lab goal from circuits 1 lab. That does not capture what the sponsor is after.

**Review by JesseTNRoberts — COMMENTED** [review]
You did not address my comment. Do not resubmit this without having first received a thorough review from your team.

**Comment by Slayde00** [comment]
Could you elaborate a little on how the purpose is not explained? For the lab comment, is that is related to the formatting or something else in the document?

**Comment by JesseTNRoberts** [comment]
The purpose of this is not to familiarize students with op amps. That is the goal of a lab. What is the purpose of this system?

**Review by JesseTNRoberts — COMMENTED** [review]
Feedback given last night

**Review by JesseTNRoberts — COMMENTED** [review]
What range of resistances does this need to accept from the breadboard? What amplitudes must the Arduino output to mimic their effect on the amplifier?

**Comment by Slayde00** [comment]
You can disregard the request. I was given feedback during the meeting!

File: Documentation/Signoff/noise cancelation.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+## Housing
+
+
+# BOM
+| Device | location | Price | Quanity | Total Cost | Source |
+|--------|----------|---------|------------|-------|--------|
+| Push Button |DigiKey| $1.85 | 1 | $1.85 | [5] |
+| Mini Speaker |DigiKey| $3.15 | 1 | $6.30 | [4]
+| SPDT switch | DigiKey| $2.82 | 1 | $5.15 | [3] |
+| 8 Ω resistor| Jameco | $1.75 | 1 | $0.80 | [2] |
```
Comment (micahr234) [inline Documentation/Signoff/noise cancelation.md]: Put the manufacture, and part number on your BOM. Do not use a reference.

**Review by micahr234 — APPROVED** [review]

**Pull request merged**